needle/core/device/remote_operations.py

```diff
@@ -155,7 +155,7 @@ def kill_proc(self, procname):
     # ==================================================================================================================
     def download(self, src, dst, recursive=False):
         """Download a file from the device."""
-        src, dst = Utils.escape_path_scp(src), Utils.escape_path_scp(dst)

```

Comment:
are you sure about this change? at the time I created the `escape_path_scp` just for dealing with scp...